### ISSUE TITLE
Visualization of Mask RCNN classification focus areas using methods like GradCAM

### ISSUE THREAD

**Issue by mist-wing2020** [matterport/Mask_RCNN#1984]
Hi, 

During my classification process, I would like to visualize the focus areas of the Mask RCNN neural network. 

I am aware of methods like GradCAM that enable the same.
I tried implementing the GradCAM basics to the mrcnn_class_logits layer, but this gave me an incorrect output. 

Is there any way to visualize where the neural network is focusing before it assigns the class to a particular instance? 

Any help or guidance will be appreciated.

**Comment by mminakshi** [comment]
Hi mist-wing2020
Could you please let me know how you implemented GradCAM here.

**Comment by Trotts** [comment]
Hi, 

I'm also wanting to implement Grad-CAM on a trained Mask-RCNN. Did anyone have any luck with this, @mist-wing2020 @mminakshi ?

**Comment by FloBr49** [comment]
Hi @mist-wing2020 @mminakshi @Trotts,

has anybody figured out how to implement GradCAM? Thank you.

**Comment by Kartiky246** [comment]
Hi @FloBr49 @Trotts @mminakshi @mist-wing2020 
has anybody figured out how to implement GradCAM? Thank you.
